.circleci/config.yml

```diff
@@ -49,17 +49,26 @@ jobs:
       - checkout
       - read_cache_linux
       - run:
-          name: "Run tests"
+          name: "Install fakeroot"
+          command: |
+            sudo apt-get update
+            sudo apt-get install fakeroot
+      - run:
+          name: "Run test"
           command: ./gradlew test
+      - store_test_results:
+          path: ~/repo/build/test-results/test/
```

Comment:
Wrong folder was being used before, moving up as well to make sure they are copied if a follow up step fails